handlers/_swagger.js

```diff
@@ -333,15 +333,16 @@ function registerRoute(app, auth, additionalMiddleware, method, path, data, allo
                 });
             }
 
-            if (responseModelValidationLevel) {
-                var responseSender = res.send;
-                res.send = function (body) {
+            var responseSender = res.send;
```

Comment:
why do we need to insert all of this code if response model validation is off?

Reply:
We intentionally included the sentBody in both validated and not-validated cases so we can read & log the body in middleware using the same logic (that is, we don't have to use one method to get a validated body and a different method to get a non-validated body)

Reply:
For the [docs in the wiki](https://github.com/BlueOakJS/blueoak-server/wiki/Handlers#response-model-validation), propose to add a bullet to the effect of:

- if set to `"warn"`...
- ...
- if response validation is enabled in any mode, the validated body is attached to the Express response object in a field named `sentBody` for use by handlers and/or middleware that are configured to execute after the response is sent.